src/traces/box/attributes.js

```diff
@@ -85,6 +85,28 @@ module.exports = {
             'For example, with 1, the whiskers are as wide as the box(es).'
         ].join(' ')
     },
+    notched: {
+        valType: 'boolean',
+        dflt: false,
+        role: 'style',
+        editType: 'calcIfAutorange',
+        description: [
+            'Determines whether or not notches should be drawn.'
+        ].join(' ')
+    },
+    notchwidth: {
+        valType: 'number',
+        min: 0,
+        max: 1,
+        dflt: 0.5,
+        role: 'style',
+        editType: 'calcIfAutorange',
+        description: [
+            'Sets the width of the notches relative to',
+            'the box\' width.',
+            'For example, with 1, the whiskers are as wide as the box(es).'
```

Comment:
It looks like Matlab and ggplot have opposite conventions about this... if I'm reading it right this is the ggplot version. I would have probably gone for the Matlab version, where `notchwidth=0` is nothing taken away, and `notchwidth=0.5` is notches that meet in the middle (actually I might have called that `1` instead... but we don't want to introduce yet ANOTHER convention). I guess I don't have that strong an opinion about it though, @etpinard or @cpsievert do you care which we use?

Reply:
Good point; I will wait for others to express their opinion. Just to have it here: the behavior (and description) of `notchwidth` is based on another, already existing attribute - `whiskerwidth`.